src/Parallel/Tags/ResourceInfo.hpp

```diff
@@ -63,5 +67,29 @@ struct SingletonInfo : db::SimpleTag {
     return resource_info.template get_singleton_info<ParallelComponent>();
   }
 };
+
+/// \ingroup ParallelGroup
+/// Tag to retrieve the ResourceInfo.
+///
+/// \details This tag is meant to be used in the GlobalCache. It can be created
+/// from options, but since it must be specially created inside Main and
+/// assigned to the GlobalCache after all the global cache tags are created from
+/// options, the return of `create_from_options` is just a default constructed
+/// Parallel::ResourceInfo.
+template <typename Metavariables>
+struct ResourceInfo : db::SimpleTag {
+  using type = Parallel::ResourceInfo<Metavariables>;
+
+  static constexpr bool pass_metavariables = true;
+
+  template <typename Metavars>
+  using option_tags = tmpl::list<Parallel::OptionTags::ResourceInfo<Metavars>>;
```

Comment:
Repeating an old comment since the threading got messed up and it got buried:

This tag isn't used below.  Is it needed to force it's inclusion in the input file for when this is initialized?  Please add a comment explaining.